<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1521230759793' stop='1521230763271'>
    <name>To test the Angular login functionality</name>
    <title>To test the Angular login functionality</title>
    <test-cases>
        <test-case start='1521230759793' status='failed' stop='1521230763270'>
            <name>Should display awesome popup on completion of multiform</name>
            <title>Should display awesome popup on completion of multiform</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='05884e67-2141-4698-95ab-fe11c4cdd0ed-attachment.png' type='image/png' size='47579'/>
            </attachments>
            <failure>
                <message>Failed: unknown error: Element &lt;input type="radio" ng-model="formData.type" value="ps" class="ng-pristine ng-valid" name="00A"> is not clickable at point (399, 307). Other element would receive the click: &lt;label>...&lt;/label>
                  (Session info: chrome=64.0.3282.186)
                  (Driver info: chromedriver=2.37.543627 (63642262d9fb93fb4ab52398be4286d844092a5e),platform=Windows NT 10.0.16299 x86_64)</message>
                <stack-trace>WebDriverError: unknown error: Element &lt;input type="radio" ng-model="formData.type" value="ps" class="ng-pristine ng-valid" name="00A"> is not clickable at point (399, 307). Other element would receive the click: &lt;label>...&lt;/label>
                  (Session info: chrome=64.0.3282.186)
                  (Driver info: chromedriver=2.37.543627 (63642262d9fb93fb4ab52398be4286d844092a5e),platform=Windows NT 10.0.16299 x86_64)
                    at Object.checkLegacyResponse (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at doSend.then.response (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:118:7)
                From: Task: WebElement.click()
                    at Driver.schedule (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at WebElement.schedule_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
                    at WebElement.click (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
                    at actionFn (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\protractor\built\element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\protractor\built\element.js:461:65)
                    at ManagedPromise.invokeCallback_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.(anonymous function).args [as click] (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (C:\Users\sandeep\Desktop\Protractor\Protractor\specs\multiform_spec.js:22:46)
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                From: Task: Run it("Should display awesome popup on completion of multiform") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at shutdownTask_.MicroTask (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:2599:53)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\sandeep\Desktop\Protractor\Protractor\specs\multiform_spec.js:11:2)
                    at addSpecsToSuite (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\sandeep\Desktop\Protractor\Protractor\specs\multiform_spec.js:3:1)
                    at Module._compile (module.js:649:30)
                    at Object.Module._extensions..js (module.js:660:10)
                    at Module.load (module.js:561:32)
                    at tryModuleLoad (module.js:501:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1521230763270' status='pending' stop='1521230763271'>
            <name>Should display error message when invalid login cradentials are entered</name>
            <title>Should display error message when invalid login cradentials are entered</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>